static/js/message_helper.js

```diff
@@ -65,6 +65,7 @@ export function process_new_message(message) {
 
             pm_conversations.process_message(message);
 
+            recent_senders.add_pm_message(message);
```

Comment:
From a naming perspective, I think maybe we should just be calling a function called `recend_senders.process_message_for_senders` unconditionally? Or we could call this `recent_senders.process_private_message`, and rename the other code path to `recent_senders.process_stream_message`.

Reply:
Did the second one.